src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -213,15 +213,15 @@ limitations under the License.
     <field name="Heating" mask="0x1"/>
     <field name="Cooling" mask="0x2"/>
     <field name="Occupancy" mask="0x4"/>
-    <field name="Schedule" mask="0x8"/>
+    <field name="ScheduleConfiguration" mask="0x8"/>
```

Comment:
Spec technically has a space in the feature name. Not sure if we care. Same for `Auto Mode`

Reply:
We do care, but we want to fix that in the spec.